test/spec/refinements.js

```diff
@@ -1,95 +1,85 @@
 'use strict';
 
-var test = require('tape');
 var _ = require('lodash');
 var algoliasearchHelper = require('../../index');
 
 var emptyClient = {};
 
-test('Adding refinments should add an entry to the refinments attribute', function(t) {
+test('Adding refinments should add an entry to the refinments attribute', function() {
   var facetName = 'facet1';
   var facetValue = '42';
 
   var helper = algoliasearchHelper(emptyClient, 'index', {
     facets: [facetName]
   });
 
-  t.ok(_.isEmpty(helper.state.facetsRefinements), 'should be empty at first');
+  expect(_.isEmpty(helper.state.facetsRefinements)).toBeTruthy();
```

Comment:
Can be stricter.

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453